packages/terra-functional-testing/tests/jest/commands/validates/element.test.js

```diff
@@ -0,0 +1,48 @@
+const { accessibility, element } = require('../../../../src/commands/validates');
+
+jest.mock('../../../../src/commands/validates/accessibility');
+
+describe('element', () => {
+  it('should run element with default options', () => {
+    global.Terra = {
+      serviceOptions: {},
+    };
+
+    element();
+
+    expect(accessibility).toHaveBeenCalledWith(undefined);
+  });
+
+  it('should run element with full options', () => {
+    global.Terra = {
+      serviceOptions: {
+        selector: 'test-selector',
+      },
+    };
+
+    element({
+      testName: 'test-name',
+      rules: { 'mock-rule': { enabled: true } },
+      selector: 'test-selector',
+      misMatchTolerance: 10,
+    });
+
+    expect(accessibility).toHaveBeenCalledWith({ 'mock-rule': { enabled: true } });
+  });
+
+  it('should run element with service selector options', () => {
```

Comment:
Is this in place for when the screenshot logic is ready to pull in?

Reply:
Yes, the implementation has the arguments extracted and ready to pass down to the screenshot logic.